node/lib/util/git_util.js

```diff
@@ -360,12 +362,18 @@ exports.push = co.wrap(function *(repo, remote, source, target, force, quiet) {
     const execString = `\
 git -C '${repo.path()}' push ${forceStr} ${remote} ${source}:${target}`;
     try {
-        let result = yield ChildProcess.exec(execString, {env : environ});
+        let result = yield ChildProcess.exec(execString, {
+            env : environ,
+            maxBuffer: EXEC_BUFFER
+        });
```

Comment:
there are plenty of similar objects, you may want to simplify them with object spread operator. For example:

```
baseOpt = {maxBuffer: 1024*1024*100}
...
let result = yield ChildProcess.exec(execString, {...baseOpt, env: environ})
```

Reply:
That's not that much shorter, so I think I'll skip it for now.